salesagility/SuiteCRM
# Fix: Report parameters Returning wrong date in set timezone (MYSQL ONLY)

## Description




This fix will make sure that no matter what timezone you have set either from the Users or if set in the config_override.php or if this is set on a server level, the dates and data with show according to the correct parameter set (Type -> Period & Value -> Today). This fix also make sure when a scheduled report is sent by email it still follows the report parameters set and will display the correct data.

This fix is for MYSQL only at the moment.

## Motivation and Context

This fix has solved data not displaying properly to a today parameter or any other with display according to your timezone
## How To Test This

1. Create closed date field for cases 
2. Set your timezone however you like.
3. Set two cases to closed date of today and one for 3am and one for 9pm
4. Create report on cases with condition Closed date field Type -> Period & Value -> Today. See data
5. set up schedule reports and send to email. See data
## Types of changes


### Final checklist
